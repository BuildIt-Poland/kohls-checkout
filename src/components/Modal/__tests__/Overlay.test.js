import React from 'react';
import { create } from 'react-test-renderer';

import Overlay from '../Overlay';

describe('COMPONENT - Modal Overlay', () => {
  it('renders correctly', () => {
    const component = create(<Overlay />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
