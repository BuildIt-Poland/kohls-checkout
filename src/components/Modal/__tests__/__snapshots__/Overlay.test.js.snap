// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Modal Overlay renders correctly 1`] = `
.c0 {
  background: rgba(0,0,0,0.6);
  height: 100%;
  left: 0;
  position: fixed;
  top: 0;
  width: 100%;
  z-index: 8000;
}

<div
  className="c0"
/>
`;
